AirCasting/Dashboard/DashboardView.swift

```diff
@@ -17,7 +17,7 @@ struct DashboardView: View {
     @EnvironmentObject var selectedSection: SelectSection
     @EnvironmentObject var averaging: AveragingService
     @State var isRefreshing: Bool = false
-
+    var urlProvider: BaseURLProvider
```

Comment:
why not private let?

Reply:
no reason why I have not used let and private in many places..